cmd/scheduleapi/handler/fixtures.go

```diff
@@ -0,0 +1,163 @@
+package handler
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+)
+
+var DynamoJSON = dynamodb.QueryOutput{
+	LastEvaluatedKey: map[string]*dynamodb.AttributeValue{
```

Comment:
I think that in practice this field only contains the hash and range key from the last evaluated document, not the entire document - I could be wrong about that though